packages/template-mrt-reference-app/app/isolation-tests.js

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
@kieran-sf in general the convention is `<filename>.test.js` is there a reason not to do that here?

Reply:
These aren't unit tests run by the CI, but functions for a new endpoint (`/isolation` similar to the `/tls` and `/cookie`) on the mrt reference app used to execute integration tests. But I have no issues changing the file name to something less confusing like `isolation.js` or `isolation-actions.js`